packages/chatbot-server-mongodb-public/tsconfig.build.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "./tsconfig.json",
-  "exclude": ["**/*.test.ts", "**/*.eval.ts"]
+  "exclude": ["**/*.test.ts"]
```

Comment:
Does this mean eval files are included in the build?

Reply:
oh that should def be added back. must have done that during debugging. nice catch.